packages/mds-audit/api.ts

```diff
@@ -647,10 +643,7 @@ function api(app: express.Express): express.Express {
           return res.status(500).send({ error: 'internal_server_error' })
         }
 
-        const {
-          start_time: [start_time],
-          end_time: [end_time]
-        } = parseRequest(req, { parser: { fn: Number } }).query('start_time', 'end_time')
+        const { start_time, end_time } = parseRequestSingle(req, { parser: Number }).query('start_time', 'end_time')
```

Comment:
With a bit of reorganization, I think there's an opportunity for a little more fluent syntax. Something like:
```ts
parseRequest(req, { parser: Number }).query('start_time', 'end_time').single()
```